source/core/ut_utils.pkb

```diff
@@ -692,5 +692,10 @@ create or replace package body ut_utils is
     return l_result;
   end;
 
+  function serialize_data (a_data clob) return clob is
```

Comment:
I would rename this function to say what it does.
Also, with CLOB you need to be careful about performance.
Using function means that the value will be copied on return.
Depending on how the function is used, maybe `in out nocopy` would be a better option?
Probably profiler could show some insight into it.